po/es.po

```diff
@@ -11467,16 +11469,16 @@ msgid ""
 "watermark color, tag:\n"
 "$(WATERMARK_COLOR)"
 msgstr ""
-"color de texto, etiqueta:\n"
+"Color de texto, etiqueta:\n"
 "$(WATERMARK_COLOR)"
 
 #: ../src/iop/watermark.c:1386
 msgid "select watermark color"
-msgstr "seleccionar color de la marca de agua"
+msgstr "Escoja el color de la marca de agua"
```

Comment:
I would say "**Elige** el color..." to maintain the same wording everywhere.

Reply:
You have to see where the message comes from in order to take into consideration the translation style.